### ISSUE TITLE
[FEATURE] Add tutorial level for first-time users

### ISSUE THREAD

**Issue by gzlzgzl** [COMP3122-2324-Project/Simpsyber#27]
Many elements in the game are unexplained. For example, it is not exactly clear how user satisfaction and the number of drivers and passengers change over time or how the player can complete specific operations like creating test cases or fixing bugs. I highly recommend adding a tutorial level that explains these concepts and guides the player through every available situation.

**Comment by Hinalai** [comment]
Your point about the lack of clarity on how certain metrics like user satisfaction and the dynamics of drivers and passengers evolve is essential. Players must understand the cause-and-effect within the game to strategize effectively.
I've also found that creating test cases and fixing bugs isn't intuitive for new players, and a tutorial could help bridge that gap. The tutorial could include interactive examples of common scenarios players might encounter and guide them through the decision-making process.
One approach could be implementing tooltip-style pop-ups that explain elements the first time the player encounters them. Alternatively, a dedicated tutorial section that players can refer to at any point during the game might be useful without hindering the flow of the gameplay.

**Comment by LAMYUHIN** [comment]
I agree with your point; the formula for user satisfaction confused me when I was first-time to play this game, and I couldn't find a detailed description or tutorial of the changes for user satisfaction in the game.

I suggest adding hints and guidelines on the result page so that the system can give hints to the players about the items with lower scores to motivate the players to keep on trying and improving.
For example, user satisfaction decreases rapidly as it takes too long to fix bugs.

**Comment by Alexachenggg** [comment]
I also agree with more tutorials for novice users. In my own experience, the first time I played the game, I didn't even know where to select the staff and what the next steps were. Until I asked someone else for help. I also think there should be a tutorial for every little step of restarting the game and giving the user hints on what to do next such as instructing them which button to press or which option to make.
